packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -394,6 +394,24 @@ class WebViewController {
     return _channel.invokeMethod("reload");
   }
 
+  /// Clears all of the following caches.
```

Comment:
This shows as the summary line in the Dartdoc site, it should be self contained, probably should just say "Clears all caches used by the [WebView]."